### PR TITLE
Creates the first version of the footer component

### PR DESCRIPTION
**Pull request by jesseez**
Creates the footer based on the mocks. Only includes the brand, the address, and the contact links. The address links to google maps, and the contact links all link to the URLs.

image

### CODE REVIEW

File: web/src/components/footer/footer.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <Box sx={footerStyles.footerContainer}>
+      <Box sx={footerStyles.brand}>
+        <Brand></Brand>
+      </Box>
+      <Location></Location>
+      <Contact></Contact>
+    </Box>
+  );
+}
```
Comment (yanbhoe) [inline web/src/components/footer/footer.js:17]: `const Footer = () => (
  <Box...>
    ...
  </Box>
);`

File: web/src/components/footer/subcomponents/brand/brand.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { Box, Typography } from "@mui/material";
+import prideLogo from "../../../../assets/images/color_logo_512.png";
+import brandStyles from "./brand.styles";
+import messages from "../../../../common/messages";
```
Comment (yanbhoe) [inline web/src/components/footer/subcomponents/brand/brand.js:4]: `import messages from "@/common/messages";`
https://stackoverflow.com/questions/42711175/what-does-the-symbol-do-in-javascript-imports

File: web/src/components/footer/subcomponents/contact/contact.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      <Typography variant="h6">Contact</Typography>
+      <Link
+        href="mailto:doyourownswing@gmail.com"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <Typography variant="body2" sx={contactStyles.link}>
+          doyourownswing@gmail.com
+        </Typography>
+      </Link>
```
Comment (yanbhoe) [inline web/src/components/footer/subcomponents/contact/contact.js:16]: `
```suggestion
const contacts = { data...}
...
  {{contacts.forEach(contact, () => 
      (<Link
        href={contact.link}
        target="_blank"
        rel="noopener noreferrer"
      >
        <Typography variant="body2" sx={contactStyles.link}>
          {contact.link}
        </Typography>
      </Link>)
 }}
```

**Review by Kuboosh21 — APPROVED** [review]

File: web/src/components/footer/footer.js (hunk 1 above)
Comment (jesseez) [reply web/src/components/footer/footer.js:17]: Looks kinda gross, ngl

File: web/src/components/footer/subcomponents/contact/contact.js (hunk 3 above)
Comment (jesseez) [reply web/src/components/footer/subcomponents/contact/contact.js]: Ty great idea, done.

**Review by yanbhoe — APPROVED** [review]

**Pull request merged**